docs/shared/packages/powerpack-conformance/powerpack-conformance-plugin.md

```diff
@@ -1,6 +1,198 @@
+| requireBuildableDependenciesForBuildableProjects | _boolean_                 | _false_ | Enable to restrict the buildable libs from importing non-buildable libraries                                                                                                                                                                   |
+| depConstraints                                   | _Array<object>_           | _[]_    | List of dependency constraints between projects                                                                                                                                                                                                |
+
+#### Dependency constraints
+
+The `depConstraints` is an array of objects representing the constraints defined between source and target projects. A
+constraint must include `sourceTag` or `allSourceTags`. The constraints are applied with **AND** logical operation - for
+a given `source` project the resulting constraints would be **all** that match its tags.
+
+| Property                 | Type            | Description                                                                        |
+| ------------------------ | --------------- | ---------------------------------------------------------------------------------- |
+| sourceTag                | _string_        | Tag that source project must contain to match the constraint                       |
+| allSourceTags            | _Array<string>_ | List of tags the source project must contain to match the constraint               |
+| onlyDependOnLibsWithTags | _Array<string>_ | The source **can depend only** on projects that contain at least one of these tags |
+| notDependOnLibsWithTags  | _Array<string>_ | The source **can not depend** on projects that contain at least one of these tags  |
+
+### Ensure Owners
+
+This rule requires every project to have an owner defined for the [`@nx/powerpack-owners` plugin](/nx-api/powerpack-owners)
+
+Set the `rule` property to: `@nx/powerpack-conformance/ensure-owners`
+
+```json {% fileName="nx.json" %}
+{
+  "conformance": {
+    "rules": [
+      {
+        "rule": "@nx/powerpack-conformance/ensure-owners"
+      }
+    ]
+  }
+}
+```
+
+## Custom Conformance Rules
+
+To write your own conformance rule, specify a relative path to a typescript file as the rule name:
+
+```json {% fileName="nx.json" %}
+{
+  "conformance": {
+    "rules": [
+      {
+        "rule": "./tools/local-conformance-rule.ts"
+      }
+    ]
+  }
+}
+```
+
+The rule definition file should look like this:
+
+```ts {% fileName="tools/local-conformance-rule.ts" %}
+import type {
+  ConformanceRule,
+  ConformanceRuleResult,
+} from '@nx/powerpack-conformance';
+
+const rule: ConformanceRule<any> = {
```

Comment:
This needs to be updated based on my PR that's under review, they should use the utility function